README.md

```diff
@@ -15,14 +15,14 @@ The EU Login module allows to authenticate against the EU Login service.
 
 ## Installation
 
-The recommended way of installing the EU Login module is via a [Composer-based workflow][2].
+The recommended way of installing the OE Auth module is via a [Composer-based workflow][2].
 
 In your Drupal project's main `composer.json` add the following dependency:
 
 ```json
 {
     "require": {
-        "openeuropa/eu_login": "dev-master"
+        "openeuropa/oe_auth": "dev-master"
```

Comment:
Let's discourage people from using dev-master, I suggest the following change:

```
"openeuropa/oe_auth": "^0.1"
```

Reply:
Nice spot. I've missed that one.